### ISSUE TITLE
API response in XML format doesn't handle properly

### ISSUE THREAD

**Issue by RuslKoval** [katalon-studio/katalon-studio#299]
#### Steps to reproduce:
1. Create REST API request GET http://petstore.swagger.io/v2/pet/1 (with header Accept:application/xml)
2. On Verification tab write script to see full xml in response body, like following:
ResponseObject response = WSResponseManager.getInstance().getCurrentResponse()
println response.getResponseText()
println response.getResponseBodyContent()
3. Test Request And Verify

#### Expected Behavior:
response.getResponseBodyContent() should return all xml content from Response Body, like

```
<Pet>
  <category>
    <id>0</id>
    <name>Cachorro</name>
  </category>
  <id>1</id>
  <name>2</name>
  <photoUrls>
    <photoUrl>null</photoUrl>
  </photoUrls>
  <status>3</status>
  <tags>
    <tag>
      <id>0</id>
      <name>string</name>
    </tag>
  </tags>
</Pet>
```

Or some other way to grad complete xml from response body.
So it is possible to verify xml values, serialize it in object, etc.

#### Actual Behavior:
Can't grab complete xml from response and work with it.
It returns just some tags.
like `<id>0</id>`

#### Screenshots / Videos: (please attach screenshots or videos if necessary to reproduce the issue)
![image](https://user-images.githubusercontent.com/23101758/76439940-d74bd600-63c5-11ea-9bef-a6aa72fb79cd.png)

**Comment by RuslKoval** [comment]
So I can't work with such response (default Katalon code snippet).
Can't use XmlParser/XmlSlurper, can't verify elements values.

**Comment by Padoplin** [comment]
This bug is fixed in version 7.3.1. Please download [here](https://github.com/katalon-studio/katalon-studio/releases).
